rules/aip0133/aip0133.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google LLC
```

Comment:
This file seems to be indented with two spaces (might be weird GitHub copy and paste though)? Please make sure you run `gofmt`.

Reply:
gofmt are executed for all the files.